### DISCUSSION TITLE
Running Phone Applications Foreground without Screen Data

### DISCUSSION THREAD

**Discussion by yikun-chi** [carissalow/rapids#201] [Q&A]
Is it possible to run phone application foreground feature without screen data? I have set the "INCLUDE_EPISODE_FEATURES" option to false, but still running into erros. It seems like the pipeline is still searching for screen data to pull in. 

Thanks!

**Comment by JulioV** [comment]
It should be possible when not computing the episode features. If you're getting errors they might be fixed once this [PR](https://github.com/carissalow/rapids/pull/198) is ready
